### DISCUSSION TITLE
CSS Implementation for border properties

### DISCUSSION THREAD

**Discussion by yashsehgal** [DesignSystemsOSS/eccentrictouch#228] [Ideas]
@WH0SUMIT will lead this discussion and each and everything that will be discussed here will be having many issues and sub-issues as well. Feel free to collaborate and share your ideas here and get some green squares as well.

**Comment by wh0sumit** [comment]
so basically we are planning to implement some border utilities to quickly style the border and border radius of an element and these utilities are very useful for images, buttons, or any other element.

Some examples of the Border Utilities are :
1- Border 
a) Border-top 
b) Border-bottom
c) Border-left 
d) Border-right
e) Border (set border on all sides of the element)
f)  Border-none (no borders)

2- Border Colors
a) Border-primary
b) Border-secondary
c) Border-invalid
d) Border-warning
d) Border-valid
e) Border-black
f)  Border-light

3- Border-Width
a) Border-1
b) Border-2
c) Border-3
d) Border-4
e) Border-5

4- Border-radius ( for element to easily round its corners )
a) Border-round
b) Border-round-top
c) Border-round-bottom
d) Border-round-left 
e) Border-round-right
f)  Border-circle

5- Border-size ( for larger or smaller rounded corners )
a) Border-rounded-0
b) Border-rounded-1
c) Border-rounded-2
d) Border-rounded-3
e) Border-rounded-4
f)  Border-rounded-5

**Comment by yashsehgal** [comment]
All these components will be listed as issues
